lnbits/core/services.py

```diff
@@ -153,9 +148,12 @@ class PaymentKwargs(TypedDict):
             extra=extra,
         )
 
-        # check_internal() returns the checking_id of the invoice we're waiting for
-        internal_checking_id = await check_internal(invoice.payment_hash, conn=conn)
-        if internal_checking_id:
+        internal_check = await check_internal(invoice.payment_hash, conn=conn)
+        if internal_check:
+
+            if not internal_check[1]:
```

Comment:
Very bad style

internal check is treaded as a bool and a tuple at the same time, and then indexed with `[1]` to make sure that the next person coding this doesn't what's going on.